recipes/lyra-v2-action-signing/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Where did this come from? Looks like there isn't anything upstream except the `pyproject.toml` (which unfortunately isn't good enough)

https://github.com/derivexyz/v2-action-signing-python/issues/11

Reply:
Right, it's also on the pypi site, but likely from the same origin. I placed an issue on their site

Reply:
> Was there any follow up on this @MementoRC ?

Yes, but upstream has been non-responsive to the issue I filed. For sake of de-cluttering, we could close this PR.